### PR TITLE
L mendoza

### PR DESCRIPTION
**Pull request by lidelmendoza**
If I delete the header file outside of the header folder the copy of the header file in the folder won't be deleted, right? Also, all I have to do in main.cpp is delete the repeating number one?

### CODE REVIEW

File: LMendoza.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+// convert miles into kilometers
+// there are 1.6 kilometers in a mile
+#include <iostream>
+using namespace std;
+#include "LMendoza.h"
```
Comment (KrumphauChicken) [inline LMendoza.h]: Circular include. A file cannot include itself.

File: main.cpp [hunk 2]
```diff
@@ -7,7 +8,7 @@ int main()
 {
 	short unsigned int choice;
 	cout << "0) Convert examples to kilo-examples.\
-	\n1) \
+	\n1) Convert miles into kilometers.
```
Comment (KrumphauChicken) [inline main.cpp]: Need a '\' on the end to continue the string across multiple lines.

**Comment by KrumphauChicken** [comment]
Correct. Removing the header that's in the top-level directory will not remove the one in the headers folder. Use `git rm`. Also, rename the header file to lowercase.

**Comment by lidelmendoza** [comment]
Okay, thanks.

**Comment by lidelmendoza** [comment]
How do I fix the conflict?

**Comment by KrumphauChicken** [comment]
In this case, just delete the >>>>, ====, and <<<< lines. They signify the changes that happened in your branch, and the changes on the same line(s) that happened in the branch you are trying to merge to since you branched (or merged last, for long-running branches).

**Comment by lidelmendoza** [comment]
I understand, thank you.

**Pull request merged**